services/maps/geocoding_service.py

```diff
@@ -0,0 +1,52 @@
+from abc import ABC, abstractmethod
+from opencage.geocoder import OpenCageGeocode, InvalidInputError, RateLimitExceededError, UnknownError
+from services.Singleton import Singleton
+
+class GeocodingRemote(ABC):
+    @abstractmethod
+    def get_information_from_address(self, address):
```

Comment:
please add the type of the variable like this `def get_information_from_address(self, address:str)`